macros/plugin.py

```diff
@@ -49,6 +49,11 @@ def on_config(self, config):
         module_reader.load_variables(self._variables, config)
 
         print("Variables:", self.variables)
+        
+        env_config = {
+            'loader': FileSystemLoader(config.get('docs_dir'))
```

Comment:
It would be good if the loader's searchpath could be overridden from the mkdocs.yml (or else from the main.py file). For example, so that templates could be placed outside of the docs dir. Otherwise, you would also need something like the mkdocs-exclude plugin to prevent the templates from being included within the built docs site.